bokeh/core/has_props.py

```diff
@@ -145,18 +145,18 @@ def __new__(meta_cls, class_name, bases, class_dict):
 
         return super().__new__(meta_cls, class_name, bases, class_dict)
 
-    def __init__(cls, _, bases, __):
+    def __init__(cls, class_name, bases, __):
         # Check for improperly redeclaring a Property attribute.
         for base in bases:
             if not issubclass(base, HasProps):
                 continue
             base_properties = base.properties()
             for attr in cls.__dict__: # we do NOT want inherited attrs here
                 if attr in base_properties:
-                    warn("Property {attr!r} in class {base.__name__} was redeclares by a class attribute "
-                         "{attr!r} in class {class_name}; it never makes sense to do this. "
-                          "Either {base.__name__}.{attr} or {class_name}.{attr} should be removed,"
-                          "or Override() should be used to change a default value of a base class property.",
+                    warn(f"Property {attr!r} in class {base.__name__} was redeclares by a class attribute "
```

Comment:
```suggestion
                    warn(f"Property {attr!r} in class {base.__name__} was redeclared by a class attribute "
```
As long as we are touching this might as well fix this grammar error too :)